Tests/instance_notifier.py

```diff
@@ -131,14 +131,11 @@ def slack_notifier(slack_token, secret_conf_path, server, user, password, build_
     if options.instance_tests:
         with open('./env_results.json', 'r') as json_file:
             env_results = json.load(json_file)
-            for env in env_results:
-                if env["Role"] == "Server Master":
-                    server = SERVER_URL.format(env["InstanceDNS"])
-                    break
+            server = SERVER_URL.format(env_results[0]["InstanceDNS"])
```

Comment:
not sure if you will get a list in case there is only one instance

Reply:
I think I will, but in any case, will run an instance build on this branch.